index.js

```diff
@@ -14,4 +16,8 @@ exports.compile = function (str, options) {
     Handlebars.registerHelper(helper, options.helpers[helper]);
   }
   return Handlebars.compile(str, options);
-}
\ No newline at end of file
+};
+exports.render = function _render(str, options, locals) {
```

Comment:
No. It is [handled automatically by jstransformer](https://github.com/jstransformers/jstransformer/blob/master/README.md#render):

> *requires the underlying transform to implement `.render` or `.compile`*

The fallback is implemented here: https://github.com/jstransformers/jstransformer/blob/master/index.js#L267-L268